library/src/commonMain/kotlin/com/eygraber/duration/extensions/Duration.kt

```diff
@@ -126,6 +126,16 @@ public inline val Double.days: Duration get() = Duration.days(this)
 public inline val Duration.inDoubleDays: Double
   get() = toDouble(DurationUnit.DAYS)
 
+/**
+ * Returns the value of this duration expressed as a [Int] number of days.
```

Comment:
```suggestion
 * Returns the value of this duration expressed as an [Int] number of days.
```

This applies to all the kdoc added.